client/src/pages/AddReview/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import propTypes from 'prop-types';
+// https://www.npmjs.com/package/react-typeform-embed
+import { ReactTypeformEmbed } from 'react-typeform-embed';
+
+import { Button } from '../../components';
+import urls from './urls';
+
+import './style.css';
+
+let typeformEmbed = null;
+const openForm = () => typeformEmbed.typeform.open();
+
+const AddReview = ({ type, sitspotId }) => (
+  <>
+    <div className="header-img header-container" />
+    <div className="add-new-review-container">
+      <h1 className="add-new-review__title">
+        We will now ask you to follow these steps to complete your
+        recommendation:
+      </h1>
+      <ol>
+        <li>
+          1) Fill in your contributor profile - only if you haven&apos;t already
+          done so.
+        </li>
+        <li>2) Provide some basic info about your sitspot.</li>
+        <li>3) Tick the sustainability credentials you observed</li>
+        <li>4) and the &quot;magical factors&quot; you liked</li>
+        <li>5) and write a brief overall review.</li>
+      </ol>
+      <p className="add-new-review__p">
+        Here are some basic principles:
+        <br />- be personal <br />- be honest <br />- be helpful <br />- if
```

Comment:
this should be a `<ul>`

Reply:
i wrote together at last min bc of a bug with the npm package, that didnt appear when we used popup mode